script.js

```diff
@@ -0,0 +1,10 @@
+function sum() {
+    var a,b;
+        var a =Number(document.getElementById("a").value)
+        var b =Number(document.getElementById("b").value)
```

Comment:
Include a space after commas, but do not group variable declarations like on line 2. They should each be on their own line. You actually did declare them on their own line below, making line 2 redundant.

In JavaScript, include a space around equals signs for readability and indent your lines the same amount. Only tab out lines when they are inside curly braces. Also, be consistent about how you end lines. You can have semicolons or you could leave them out. Name your variables something descriptive so people know what they represent.